src/main/ngapp/accio/src/app/editor/magic-wand.service.ts

```diff
@@ -7,95 +7,116 @@ import { SetOperator } from './set-operator';
 })
 export class MagicWandService {
   /**4-Way floodfill (left, right, up, down):
-   * Return set of coordinates(formatted as a 1-D array).
+   * @returns {Set<number>} set of coordinates(formatted as a 1-D array).
    * Coordinates correspond to pixels considered part of the mask.
    */
   /* tslint:disable */
   floodfill(imgData: ImageData, xCoord: number, yCoord: number,
       tolerance: number): Set<number> {
-    /**Store a queue of coords for pixels that we need to visit in "visit".
-     * Store already-visited pixels in "visited" as index formatted numbers
-     * (as opposed to coord format; for Set funcs).
-     */
+    // Stores a queue of coords for pixels that we need to visit in "visit".
     const visit: Array<Array<number>> = new Array();
+    // Stores already-visited pixels in "visited" as index formatted numbers
+    // (as opposed to coord format; for Set funcs).
     const visited: Set<number> = new Set();
-    // Use a set for mask; mainly do iter and set operations on masks
+    // Uses a set for mask; mainly do iter and set operations on masks...
     const mask: Set<number> = new Set();
-    // Represent [R,G,B,A] attributes of initial pixel
+    // Represents [R,G,B] attributes of initial pixel.
     const originalPixel: Array<number> =
-        this.dataArrayToRgba(imgData, xCoord, yCoord);
+        this.dataArrayToRgb(imgData, xCoord, yCoord);
 
     visit.push([xCoord, yCoord]);
-    // Convert [x,y] format coord to 1-D equivalent of imgData.data (DataArray)
+    // Converts [x,y] format coord to 1-D equivalent of
+    // imgData.data (DataArray).
     const indexAsDataArray: number =
         this.coordToDataArrayIndex(xCoord, yCoord, imgData.width);
     visited.add(indexAsDataArray);
 
-    // Loop until no more adjacent pixels within tolerance level
+    // Loops until no more adjacent pixels within tolerance level.
     while (visit.length !== 0) {
       const coord: Array<number> = visit.pop();
-      // Unpack coord
+      // Unpacks coord
       const x: number = coord[0];
       const y: number = coord[1];
 
-      // Operational part of while-loop
+      // Operational part of while-loop...
```

Comment:
nit: three periods. Should only be one as you want full sentences.  Here and below.